### PR TITLE
printing licenses violation to cli-output

### PR DESCRIPTION
**Pull request by itai1357**
**By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

## Description
* print licenses violation to cli output
* adjust relevant tests

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my feature, policy, or fix is effective and works
- [x] New and existing tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: checkov/sca_package/output.py [hunk 1]
```diff
@@ -239,61 +235,106 @@ def create_cli_output(fixable=True, *cve_records: List[Record]) -> str:
...
+
+                    if record.check_result["result"] == CheckResult.SKIPPED:
+                        cve_count.skipped += 1
+                        continue
+                    else:
+                        cve_count.to_fix += 1
+
+                    # best way to dynamically access an class instance attribute
+                    severity_str = record.severity.name.lower()
+                    setattr(cve_count, severity_str, getattr(cve_count, severity_str) + 1)
```
Comment (gruebel) [inline checkov/sca_package/output.py:257]: is this needed? I don't see anywhere being used.

**Review by gruebel — COMMENTED** [review]
nice, looks good. found only one weird thing.

File: checkov/sca_package/output.py (hunk 1 above)
Comment (itai1357) [reply checkov/sca_package/output.py:257]: no idea. not my code (I just did indentation )

**Review by gruebel — APPROVED** [review]
🚀

**Pull request merged**